schemas/com.google.tag-manager.server-side/view_item_list/jsonschema/1-0-0

```diff
@@ -0,0 +1,239 @@
+                        "type": ["string", "null"],
+                        "maxLength": 4096
+                },
+                "items": {
+                        "description": "The items for the event.",
+                        "type": "array",
+                        "items": {
+                                "description": "An item object.",
+                                "type": "object",
+                                "oneOf": [
+                                        {
+                                                "description": "An item with item_id as required.",
+                                                "properties": {
+                                                        "item_id": {
+                                                                "description": "The ID of the item.",
+                                                                "type": "string",
+                                                                "maxLength": 500
+                                                        },
+                                                        "item_name": {
+                                                                "description": "The name of the item.",
+                                                                "type": ["string", "null"],
+                                                                "maxLength": 500
+                                                        },
+                                                        "affiliation": {
+                                                                "description": "A product affiliation to designate a supplying company or brick and mortar store location",
+                                                                "type": ["string", "null"],
+                                                                "maxLength": 4096
+                                                        },
+                                                        "coupon": {
+                                                                "description": "The coupon name or code associated with the item.",
+                                                                "type": ["string", "null"],
+                                                                "maxLength": 4096
+                                                        },
+                                                        "currency": {
+                                                                "description": "The currency, in 3-letter ISO 4217 format.",
+                                                                "type": ["string", "null"],
+                                                                "minLength": 3,
+                                                                "maxLength": 3
+                                                        },
+                                                        "discount": {
```

Comment:
Thank you @istreeter ! Indeed the discount is meant to be positive. My initial concern to not add zero as minimum was whether it may be tracked as a negative number (as sometimes depicted in marketplaces). I could not find docs as to what that is meant to be, or how google will handle it, but judging from the example value shown, indeed discount is probably meant to be positive. 
Force-pushed with adding `"minimum": 0` to all server-side schemas with `items`, namely:
`add_payment_info`, `add_to_cart`, `add_to_wishlist`, `begin_checkout`, `purchase`, `refund`, `view_item`, `view_item_list`, `view_search_results`.